src/master-playlist-controller.js

```diff
@@ -1002,7 +1002,12 @@ export class MasterPlaylistController extends videojs.EventTarget {
       };
 
       if (variant.attributes && variant.attributes.CODECS) {
-        variantCodecs = parseCodecs(variant.attributes.CODECS);
+        let codecString = variant.attributes.CODECS;
+
+        variantCodecs = parseCodecs(codecString);
+        if (!MediaSource.isTypeSupported('video/mp4; codecs="' + codecString + '"')) {
```

Comment:
I just noticed that adding this MediaSource.isTypeSupported -check leads problems with HLS playlists using EXT-X-VERSION: 3 that is default version in Wowza Streaming Engine 4.x

At least on Chrome MediaSource.isTypeSupported returns false for codec "avc1.66.31,mp4a.40.2" although it's playing normally. This causes excluding all playlist files.

This problem seems to be fixed if I change EXT-X-VERSION to 4 because then it reports codecs in this format: "avc1.42001f,mp4a.40.2" and MediaSource.isTypeSupported returns true.

About changing EXT-X version https://www.wowza.com/forums/content.php?380-How-to-change-the-EXT-X-VERSION-for-Apple-HTTP-Live-Streaming

Here is examples of playlist used
```
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-STREAM-INF:BANDWIDTH=2289146,CODECS="avc1.77.32,mp4a.40.2",RESOLUTION=1280x720
chunklist_w1345153233_b2289146.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=1510142,CODECS="avc1.66.31,mp4a.40.2",RESOLUTION=960x540
chunklist_w1345153233_b1390142.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=740142,CODECS="avc1.66.30,mp4a.40.2",RESOLUTION=640x360
chunklist_w1345153233_b690142.m3u8
```

```
#EXTM3U
#EXT-X-VERSION:4
#EXT-X-STREAM-INF:BANDWIDTH=3354411,CODECS="avc1.4d0020,mp4a.40.2",RESOLUTION=1280x720
chunklist_w663865548_b3354411.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=1510164,CODECS="avc1.42001f,mp4a.40.2",RESOLUTION=960x540
chunklist_w663865548_b1390164.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=740164,CODECS="avc1.42001e,mp4a.40.2",RESOLUTION=640x360
chunklist_w663865548_b690164.m3u8
```